internal/integration/integration_test.go

```diff
@@ -542,3 +541,11 @@ func (r *rrw) ReadRevisions(ctx context.Context) (migrate.Revisions, error) {
 func (r *rrw) clean() {
 	*r = rrw(migrate.Revisions{})
 }
+
+func compileCLI(t *testing.T) (string, error) {
```

Comment:
```suggestion
func buildCmd(t *testing.T) (string, error) {
```
A suggestion, no need to rename if you don't like it.